### ISSUE TITLE
Problem when running the app locally

### ISSUE THREAD

**Issue by cmartinotti** [napoles-uach/streamlit_network#1]
This might be an ingenuous problem, but when I run the app locally  the graph displays in a new google chrome tab, instead of displaying it in the app (as instead happens when I run the app from the link https://share.streamlit.io/napoles-uach/streamlit_network/main/app.py ).
This is obviously very inconvenient, is there a way to correct the behaviour to the expected one?

Thank you for the awesome job by the way!

**Comment by napoles-uach** [comment]
Hi @cmartinotti, thanks for reporting this. That behavior is not expected, but it seems to occur in certain circumstances. Could you try to do something like 
```
net.Network(notebook=True, ...)
```
please?
Let me know if it works.
Regards!!

**Comment by cmartinotti** [comment]
Hello again, something weird is going on because this morning (I'm in AU) I switched on the computer and now running the same code the behaviour seems fixed. Yesterday it seems that just calling g.show() would auto open the file in the browser. On the flipside I'm getting a message on the console that I can't remember getting yesterday:
```
/usr/bin/www-browser: line 108: /mnt/c/Windows/System32/reg.exe: Input/output error
/usr/bin/www-browser: line 98: /mnt/c/Windows/System32/chcp.com: Input/output error
/usr/bin/www-browser: line 103: /mnt/c/Windows/System32/WindowsPowerShell/v1.0/powershell.exe: Input/output error
/usr/bin/www-browser: line 98: /mnt/c/Windows/System32/chcp.com: Input/output error
/usr/bin/www-browser: line 108: /mnt/c/Windows/System32/reg.exe: Input/output error
/usr/bin/www-browser: line 98: /mnt/c/Windows/System32/chcp.com: Input/output error
/usr/bin/www-browser: line 103: /mnt/c/Windows/System32/WindowsPowerShell/v1.0/powershell.exe: Input/output error
/usr/bin/www-browser: line 98: /mnt/c/Windows/System32/chcp.com: Input/output error
```

To be more complete I am now using this  code to test :
```
    g=net.Network(height='800px', width='800px')
    g.add_node(1)
    g.add_node(2)
    g.add_node(3)
    g.add_edge(1,2)
    g.add_edge(2,3) 
    import streamlit.components.v1 as components
    # THIS LAST PART WAS THE TROUBLESOME  PART
    g.show('example.html')
    HtmlFile = open("example.html", 'r', encoding='utf-8')
    source_code = HtmlFile.read() 
    components.html(source_code, height = 1200,width=1000)
```

If I change the last 4 lines with this (from this example https://towardsdatascience.com/making-network-graphs-interactive-with-python-and-pyvis-b754c22c270 ):
```
from IPython.core.display import display, HTML
display(HTML('example.html'))
```
That doesn't work still and it display nothing.
If I try to change the last 4 lines with the following (from this example https://discuss.streamlit.io/t/new-pyvis-component-for-graphs/11335):
```
from stvis import pv_static
pv_static(g)
```
It works fine again (yesterday it was popping a new window).

Lastly I try in a Jupyter notebook with notebook=True it works fine (and was working fine even yesterday, no spwaning of new windows), but I discovered that using the VSCode Jupyter notebook plugin with Notebook=True the graph is not displayed and instead only a white rectangle appears.

**Comment by napoles-uach** [comment]
Hi @cmartinotti, thanks a lot for your feedback.  I think I've found a solution for this issue, but I'll need to update the component stvis. I'll let you know as soon as this is ready, hopefully in a few days as I'm busy now. Please stay tuned!
Regards!

**Comment by napoles-uach** [comment]
Hi @cmartinotti, I've updated the stvis library. Please try installing with `pip install stvis==0.0.2` and use the following code to test. 
```python
from pyvis import network as net
import streamlit as st
from stvis import pv_static

g=net.Network(height='500px', width='500px',heading='')
g.add_node(1)
g.add_node(2)
g.add_node(3)
g.add_edge(1,2)
g.add_edge(2,3) 

pv_static(g)
```

Let me know how it goes!

**Comment by cmartinotti** [comment]
I just tried and it seems to work! Now, weirdly enough as I mentioned the old version seems to have fixed itself out of the blue, may I ask what was the issue? Thank you so much for your time!

**Comment by napoles-uach** [comment]
Sure, it was a call to a method of pyvis that by itself creates an HTML file, and I realized that it was kind of redundant. In some way, the HTML file was called two times, one from streamlit and the second by the method `show` which at first I didn't expect to do. So I had to review the functions in the pyvis source code and use the one that just writes the HTML file, but that doesn't attempt to open it (which is the method write_html).
